packages/contracts/src/abis/erc20.ts

```diff
@@ -1,7 +1,7 @@
 /**
  * [ERC-20 Token Standard](https://ethereum.org/en/developers/docs/standards/tokens/erc-20)
  */
-export const erc20ABI = [
+export const ERC20_ABI = [
```

Comment:
- investor-foxy is also redeclaring its own, can we leverage this guy there as well?

https://github.com/shapeshift/web/blob/72aed01fb9be296e39fa350e8d195d3d1b6d12b5/src/lib/investor/investor-foxy/abi/erc20-abi.ts#L3

- perhaps even better, do we even need to redeclare the erc20 ABI altogether or can we import from viem? e.g:

https://github.com/shapeshift/web/blob/72aed01fb9be296e39fa350e8d195d3d1b6d12b5/src/pages/RFOX/hooks/useStakingBalanceOfQuery.ts#L7

Reply:
Oooh yes actually lets pull in viem stuff wherever possible

Reply:
Still an occurence in `packages/chain-adapters/src/evm/erc20Abi.json` if we want to clean that up 🙏🏽